src/test/java/io/zipcoder/interfaces/ClassRoomTest.java

```diff
@@ -0,0 +1,37 @@
+package io.zipcoder.interfaces;
+
+import java.util.HashMap;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ClassRoomTest {
+	
+	@Test
+	
+	public void getRosterTest(){
+		
+		//Given
+		HashMap<String, Person> testRoster = new HashMap<String, Person>();
```

Comment:
Line 19 and 15 can be re-written as
`HashMap<String, Person> testRoster = ClassRoom.getRoster()`
this is the _Given_ clause of the test.